saifa/umbrel-app.yml

```diff
@@ -0,0 +1,36 @@
+manifestVersion: 1
+id: saifa
+category: finance,development tool
```

Comment:
We currently only support one category (may change in the future). I'd suggest `bitcoin` for now so that it will show up with related apps.